superset/multi_tenant.py

```diff
@@ -0,0 +1,37 @@
+from flask_appbuilder.security.sqla.manager import SecurityManager
+from flask_appbuilder.security.sqla.models import User
+from sqlalchemy import Column, Integer, ForeignKey, String, Sequence, Table
+from sqlalchemy.orm import relationship, backref
+from flask_appbuilder import Model
+from flask_appbuilder.security.views import UserDBModelView
+from flask_babel import lazy_gettext
+
```

Comment:
NIT: our linter / PEP8 likes 2 empty lines before method and class definition. To lint locally, run `flake8 superset/` from the root of the repo